frontend/src/components/Collections/components/Grid/components/DatasetsGrid/index.tsx

```diff
@@ -56,12 +40,12 @@ const DatasetsGrid: FC<Props> = ({
       <tbody>
         {sortByCreatedAtAscending(datasets).map((dataset) => (
           <DatasetRow
+            visibility={visibility}
```

Comment:
`visibility` and `accessType` need to be passed to each `DatasetRow` now, so we can hide/show action buttons based on collection's visibility and the user's access type (read/write)